local-cli/generator-ios/templates/app/AppDelegate.m

```diff
@@ -47,6 +47,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                       moduleName:@"<%= name %>"
                                                initialProperties:nil
                                                    launchOptions:launchOptions];
+  rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
```

Comment:
It is more terse, but that's not the point :-)  If you want to actually customize the colour, you need to see the Objective-C syntax, and hiding it all behind whiteColor was explicitly not done for that reason.